src/models/response.rs

```diff
@@ -0,0 +1,46 @@
+use actix_web::HttpResponse;
+
+#[derive(serde::Serialize)]
```

Comment:
very nice

Reply:
perhaps data could be an option since some API calls don't necessarily have data to return. But that's not too important